src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs

```diff
@@ -285,15 +183,44 @@ public void ClearAccessibilityRelations()
         /// <summary>
         /// Gets accessibility collection connected with the current object.
         /// </summary>
-        /// <returns>AddressCollection</returns>
+        /// <returns>A dictionary mapping a relation type to a set of objects in that relation</returns>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public AddressCollection GetAccessibilityRelations()
+        public Dictionary<AccessibilityRelationType, List<View>> GetAccessibilityRelations()
         {
-            var result = new AddressCollection(Interop.ControlDevel.DaliToolkitDevelControlNewGetAccessibilityRelations(SwigCPtr));
+            var list = new List<KeyValuePair<int, IntPtr>>();
+            var listHandle = GCHandle.Alloc(list);
+            var callback = new Interop.ControlDevel.GetAccessibilityRelationsCallback(GetAccessibilityRelationsCallback);
+
+            Interop.ControlDevel.DaliToolkitDevelControlGetAccessibilityRelations(SwigCPtr, callback, GCHandle.ToIntPtr(listHandle));
+            listHandle.Free();
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+
+            var result = new Dictionary<AccessibilityRelationType, List<View>>();
+
+            foreach (var pair in list)
+            {
+                var key = (AccessibilityRelationType)pair.Key;
+                var value = this.GetInstanceSafely<View>(pair.Value);
```

Comment:
Wow! You don't miss this point!.
`pair.value` mallocated inside of csharp-binder side. But due to this instance getter, It will be safe in memory leak.
LGTM